### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by abdulhamiid**
Hi, 👋

I have implemented the project requirements given in the below link:

https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md

## Screenshots
vetL
vetL2
vetL3
vetL4

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -36,4 +36,26 @@ ALTER TABLE animals
...
+species_id INT NOT NULL,
+vets_id INT NOT NULL,
+CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADE,
+CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id) ON DELETE CASCADE
+);
+
+CREATE TABLE visits (
+animals_id INT NOT NULL,
+vets_id INT NOT NULL,
+CONSTRAINT fk_animals FOREIGN KEY(animals_id) REFERENCES animals(id) ON DELETE CASCADE,
```
Comment (RolandM99) [inline schema.sql:59]: - [ ] Kindly, add the column `visit_date` with the data type `date` in the table `visits` as requested in the project requirements

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @abdulhamiid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (abdulhamiid) [reply schema.sql:59]: Yeah, thanks I forgot to add the command I used to create the 'visit_date' column

**Review by FrederickMih — APPROVED** [review]
Hi @abdulhamiid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**